learn/settings.py

```diff
@@ -215,6 +215,11 @@
 
 HUBSPOT_API_KEY = env('HUBSPOT_API_KEY')
 
+# SLACK
+# ------------------------------------------------------------------------------
+
+SLACK_USER_OAUTH_TOKEN = env('SLACK_USER_OAUTH_TOKEN')
```

Comment:
Should we add a comment to explain that this is Rocket's admin user OAuth token?